src/apps/watchfaces/OswAppWatchface.cpp

```diff
@@ -140,6 +144,8 @@ void OswAppWatchface::handleButtonDefaults() {
         OswHal::getInstance()->increaseBrightness(25);
     if (OswHal::getInstance()->btnHasGoneDown(BUTTON_2))
         OswHal::getInstance()->decreaseBrightness(25);
+    if (OswHal::getInstance()->btnIsDoubleClick(BUTTON_2))
```

Comment:
I think there will be problems with users trying to decrease their screen brightness and accidentally overwriting their favorite (default) watchface.

Reply:
I think there is the problem, that users tend to really spam the brightness control buttons whenever they _quickly_ want to change it!

Ideas:

1. Try to use long press instead for the default watchface setting. There is still no visual indicator, about what was done, but still better than colliding controls!
2. #266 wanted to implement that anyways, right? Sou the watchface could add an additional multi-button mode for that. Whenever the user switches modes, a little indicator icon will show up temporarily to communicate which mode the buttons are an - this also does not collide with the brightness control...